custom_components/smartthinq_sensors/wideq/ac.py

```diff
@@ -882,11 +897,21 @@ def duct_zones_type(self):
             return None
         return self.to_int_or_none(self._data.get(DUCT_ZONE_V1_TYPE))
 
+    @property
+    def lighting_display(self):
+        key = self._get_state_key(STATE_LIGHTING_DISPLAY)
+        if self._data.get(key) == self.to_int_or_none(LIGHTING_DISPLAY_ON):
+            value = True
+        else:
+            value = False
+        return self._update_feature(FEAT_LIGHTING_DISPLAY, value, False)
```

Comment:
Here is better to check if light status is supported, so that device that not support it will not have the feature and the switch:

```
        if (value := self.to_int_or_none(self._data.get(key))) is None:
            return None
        return self._update_feature(FEAT_LIGHTING_DISPLAY, value==LIGHTING_DISPLAY_ON, False)
```

Reply:
Done!